shared/package.json

```
@@ -1,20 +1,20 @@
 {
-  "name": "shared",
-  "private": true,
-  "version": "0.0.1",
-  "engines": {
-    "node": ">=14 <15",
-    "npm": ">=6 <7"
-  },
-  "scripts": {
-    "lint:js": "npx eslint \"src/**/*.ts\"",
-    "lint": "npm run lint:js",
-    "build:ts": "tsc && tscpaths -p tsconfig.json --src src --out build",
-    "build": "npm run build:ts && cp package.json build"
-  },
-  "devDependencies": {
-    "tscpaths": "0.0.9",
-    "typescript": "4.3.5"
-  },
-  "dependencies": {}
+    "name": "shared",
+    "private": true,
+    "version": "0.0.1",
+    "engines": {
+        "node": ">=14 <15",
+        "npm": ">=6 <7"
+    },
+    "scripts": {
+        "lint:js": "npx eslint \"src/**/*.ts\"",
+        "lint": "npm run lint:js",
+        "build:ts": "tsc && tscpaths -p tsconfig.json --src src --out build",
+        "build": "npm run build:ts && copy package.json build"
```

Please revert back to `cp` command. 
```suggestion
      "build": "npm run build:ts && cp package.json build"
```